src/user-research/choosing-and-using-analysis-and-synthesis-methods.md

```diff
@@ -26,6 +32,8 @@ In doing analysis and synthesis we want:
 * to make analysis and synthesis a team sport - so our colleagues get involved and become advocates for our findings
```

Comment:
@johnwaterworth These two points seem to say the same thing, I was going to suggest merging them, but on a closer look I think that for one you want to make a point of justifying next steps and for the other of being able to backtrack decisions (so it is about documenting). So, here I am going to suggest a way of changing them slightly so they are clearly different instead of appearing to be similar.

The two points I refer to are these (I just could not add a comment in any of those lines in Github!):
"-findings and next steps we can justify - as we are often challenged to explain why we have come to those conclusions
-findings we can trace back to our research data - as we are often asked to show the evidence for our findings and where our ideas came from"

This is the suggested edit:
"-next steps we can justify - as we are often challenged to explain why we have come to those conclusions
  -findings we can trace back to our research data - as we are often asked to show the evidence for our findings and where our ideas came from. We document our decisions during analysis and synthesis."

Reply:
I agree. The two points are written in a way that overlaps. I've included some, hopefully, imporved wording in the new pull request.